### ISSUE TITLE
rearrange icons for puzzle 4

### ISSUE THREAD

**Issue by cschois1** [iamsammak/thehuntgame#145]
Rearrange icons such that the bok-choy, steaming pot, and green onions are not disabled by the strong magnet and not in a line (either horizontal, vertical, or diagonal). Switching the tree icon with the steaming pot icon should suffice, since the answer would still be 3,4,9 and then the tree (6) would be disabled by the strong magnet.

**Comment by iamsammak** [comment]
If we switch "Tree" with "Steaming Pot" this is changing the answer

**Comment by iamsammak** [comment]
The better question is what are the correct answer icons? @cschois1

**Comment by iamsammak** [comment]
Cai = Bok Choy
Shu = Tree
Cong = Green Onion
^thats what I thought the answer was

**Comment by cschois1** [comment]
OOOHH. Haha I didn't realize that the Mandarin pinyin for 述 is like tree! In Cantonese, it's less similar. So the actual character is 述, tree is 树, but "cooked/ready/mature" is 熟. The Mandarin for all of these are almost identical, but the Cantonese isn't (enter the characters here to hear it in Cantonese: https://www.chineseconverter.com/cantonesetools/en/cantonese-text-to-sound).

I think this would actually be helpful if we switched the steaming pot (which is the closest thing I found to "cooked/ready") with the tree--that way the strong magnet would disable the key icon that Mandarin speakers would think is the solution, prompting them to pay closer attention to the fact that my family speaks Cantonese instead of Mandarin.
